lnwire/open_channel.go

```diff
@@ -141,6 +141,13 @@ type OpenChannel struct {
 	// type.
 	LeaseExpiry *LeaseExpiry
 
+	// LocalNonce is an optional field that transmits the
+	// local/verification nonce for a party. This nonce will be used to
+	// verify the very first commitment transaction signature.  This will
+	// only be populated if the simple taproot channels type was
+	// negotiated.
+	LocalNonce *Musig2Nonce
```

Comment:
I think this is out of sync with the bolt proposal? There it is specified that we add both a verification nonce _and_ a signing nonce to this message and others

Reply:
Commented below, this integrates the nonce modifications proposed by the LDK team.